### PR TITLE
Update logistic.md

### PR DESCRIPTION
**Pull request by colbec**
Various little syntax changes to suit version 1.0; a workaround for the rounding of the loss since macro @sprintf is currently not working.

### CODE REVIEW

File: docs/src/logistic.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 
 ```julia
-using Distributions
+using Distributions, TensorFlow, PyCall, MacroTools
```
Comment (malmaud) [inline docs/src/logistic.md]: Why do you need PyCall and MacroTools?

File: docs/src/logistic.md [hunk 2]
```diff
@@ -39,10 +39,11 @@ saver = train.Saver()
 # Run training
 run(sess, global_variables_initializer())
 checkpoint_path = mktempdir()
-@info("Checkpoint files saved in $checkpoint_path")
+@info "Checkpoint files saved in $checkpoint_path"
```
Comment (malmaud) [inline docs/src/logistic.md]: Why get rid of the parens?

File: docs/src/logistic.md [hunk 3]
```diff
@@ -39,10 +39,11 @@ saver = train.Saver()
...
 for epoch in 1:100
     cur_loss, _ = run(sess, (Loss, minimize_op), Dict(X=>x, Y_obs=>y))
-    println(@sprintf("Current loss is %.2f.", cur_loss))
+    cur_loss_rounded = round(cur_loss,digits=2)
+    println("Current loss is $cur_loss_rounded.")
```
Comment (malmaud) [inline docs/src/logistic.md]: What's wrong with `@sprintf`? Did Julia 1.0 change its API or something?

Comment (colbec) [reply docs/src/logistic.md]: Ditto again, it would not run, so I looked and found an old discussion about sprintf and how the community was looking for some volunteer to resolve the outstanding issues. Perhaps these are now resolved. I did a `? sprintf` on my machine and found nothing of use. So I worked around it using functionality that appeared to be available. Let me know if I can help by further digging.

File: docs/src/logistic.md (hunk 1 above)
Comment (colbec) [reply docs/src/logistic.md]: I just re-tested the old code and now with PyCall and MacroTools not called for the code works. I'm not able to explain why the situation was different the first time.

File: docs/src/logistic.md (hunk 2 above)
Comment (colbec) [reply docs/src/logistic.md]: On second test the code works with or without parens.

File: docs/src/logistic.md (hunk 3 above)
Comment (malmaud) [reply docs/src/logistic.md]: It's moved to `Printf` standard library module, apparently.

**Comment by malmaud** [comment]
I just tweaked a few things; should be g2g now.
